libraries/AP_GPS/AP_GPS.h

```diff
@@ -310,7 +310,8 @@ class AP_GPS
     }
 
     // the expected lag (in seconds) in the position and velocity readings from the gps
-    float get_lag() const { return 0.2f; }
+    float get_lag(uint8_t instance) const;
```

Comment:
I had a go at it, but ran into compiler errors associated with use of the override attribute on the get_lag function that I did not understand, so will tackle that separate to this PR.